### ISSUE TITLE
Error in showing Adjust Project Percentages 

### ISSUE THREAD

**Issue by maryjaf** [GeneralMagicio/pw-frontend#89]
After voting and tapping on "finish ranking", in "Adjust Project Percentages" page the all percentages are shown zero

<img width="1436" alt="image" src="https://github.com/GeneralMagicio/pw-frontend/assets/111529185/2f25896a-66a6-45bf-9e36-5b5e6f32e772">

<img width="1375" alt="image" src="https://github.com/GeneralMagicio/pw-frontend/assets/111529185/10a6cde8-e7d4-4453-acd7-490bb4a7f71d">

**Comment by maryjaf** [comment]
This issue hasn't been fixed yet , plz check @mmahdigh 
<img width="1406" alt="image" src="https://github.com/GeneralMagicio/pw-frontend/assets/111529185/00b253fe-a837-4553-a128-601056ed5625">

**Comment by mmahdigh** [comment]
Ah, I got it. The reason they are all 0 here is different from the initial bug that caused the issue in the first image.

You're seeing percentages in a collection that its total share is 0%. See the `Budget Allocation (Total: 0.00%)`.

This is more of a design problem that we are already aware of. Whether we need to let users vote in collections that have 0% share or not, as it's pointless anyway.

**Comment by maryjaf** [comment]
> This is more of a design problem that we are already aware of. Whether we need to let users vote in collections that have 0% share or not, as it's pointless anyway.

Do we have any plan to change this ? @MoeNick

**Comment by maryjaf** [comment]
There is another issue in this page plz check @mmahdigh  
When I change the percentage the ordering based on the percentage isn't changed correctly

<img width="1105" alt="image" src="https://github.com/GeneralMagicio/pw-frontend/assets/111529185/30489175-757f-40f4-96a5-c35806639088">

**Comment by mmahdigh** [comment]
> There is another issue in this page plz check @mmahdigh When I change the percentage the ordering based on the percentage isn't changed correctly

Is the ordering wrong after refreshing the page or clicking "Update ranks"?  Because we currently don't change the order after updating ranks but only after doing a refresh.

**Comment by maryjaf** [comment]
> > There is another issue in this page plz check @mmahdigh When I change the percentage the ordering based on the percentage isn't changed correctly
> 
> Is the ordering wrong after refreshing the page or clicking "Update ranks"? Because we currently don't change the order after updating ranks but only after doing a refresh.

Yeah by refreshing the ordering problem will be resolved
